src/DynamoCore/Nodes/DummyNode.cs

```diff
@@ -5,11 +5,12 @@
 using System.Xml;
 using Autodesk.DesignScript.Runtime;
 using Dynamo.Models;
+using Dynamo.Properties;
 
 namespace DSCoreNodesUI
 {
     [NodeName("Legacy Node")]
-    [NodeDescription("This is an obsolete node")]
+    [NodeDescription("DummyNode", typeof(Resources))]
```

Comment:
Hi @zora-wang, the term `DummyNode` here isn't very descriptive. Localization team would look at this and not sure what to do with it, but if we name it as `DummyNodeNodeDescription`, then it becomes clear, it is *a description for the dummy node*. Does that make sense? So please go through all the `NodeDescription` tags in the solution, and then make sure they all follow this format: `NodeXyzNodeDescription`.

Thanks!